mev_inspect/prices.py

```diff
@@ -20,7 +20,32 @@ def fetch_prices() -> List[Price]:
         price_time_series = price_data["prices"]
 
         for entry in price_time_series:
-            timestamp = dt.fromtimestamp(entry[0] / 100)
+            timestamp = dt.fromtimestamp(entry[0] / 1000)
+            token_price = entry[1]
+            prices.append(
+                Price(
+                    timestamp=timestamp,
+                    usd_price=token_price,
+                    token_address=token_address,
+                )
+            )
+
+    return prices
+
+
+def fetch_prices_range(start: int, end: int) -> List[Price]:
```

Comment:
even though this is a "quick" thing, worth the extra 5 mins to make it share logic w/ the one above which is almost identical

Reply:
like defining a shared function?

Reply:
yeah sharing the logic where it's the same

Reply:
or at least the piece to go `prices_data` => `List[Price]`